src/dali/abstract_pair_converter_plugin.py

```diff
@@ -87,3 +117,109 @@ def get_conversion_rate(self, timestamp: datetime, from_asset: str, to_asset: st
             )
 
         return result
+
+    def _build_fiat_list(self) -> None:
+        try:
+            response: Response = self.__session.get(_EXCHANGE_SYMBOLS, timeout=self.__TIMEOUT)
+            # {
+            #     'motd':
+            #         {
+            #             'msg': 'If you or your company ...',
+            #             'url': 'https://exchangerate.host/#/donate'
+            #         },
+            #     'success': True,
+            #     'symbols':
+            #         {
+            #             'AED':
+            #                 {
+            #                     'description': 'United Arab Emirates Dirham',
+            #                     'code': 'AED'
+            #                 },
+            #             ...
+            #         }
+            # }
+            data: Any = response.json()
+            if data[_SUCCESS]:
+                self.__fiat_list = [fiat_iso for fiat_iso in data[_SYMBOLS] if fiat_iso != "BTC"]
```

Comment:
Is BTC the only one that needs to be filtered out? Are there any other crypto tickers among these values?

Reply:
The documentation is a little sparse on exchangerates.host. That's the only one I found, but it could introduce another major crypto asset later like ETH, so we should be on the look out for a suitable replacement.

currency.com is supported by CCXT and trades synthetic FX that seems to be pretty accurate, but, you know, it is synthetic pricing. We'll have to keep looking.

I'm honestly surprised no one has combined FX with crypto, but I'm sure there are regulations preventing it somewhere.

Reply:
Ok, sounds good, I was just curious. We can add more tickers as the need arises.